charts/threat-protection/values.yaml

```diff
@@ -0,0 +1,119 @@
+threat_detection: 
+  aktoApiSecurityDetection:
```

Comment:
These values are not being used anywhere. **This helm chart will not work.** Please correct the same.